build/configs/stm32l4r9ai-disco/hello/defconfig

```diff
@@ -682,7 +682,7 @@ CONFIG_FB_OVERLAY=y
 # CONFIG_WATCHDOG is not set
 # CONFIG_TIMER is not set
 # CONFIG_ANALOG is not set
-# CONFIG_DRIVERS_KERNEL_TEST is not set
+# CONFIG_DRIVERS_OS_API_TEST is not set
```

Comment:
We prefer splitting a commit for this. Because this makes difficult to see real change of this commit.